code/lib/builder-vite/README.md

```diff
@@ -99,65 +65,48 @@ For other details about the differences between vite and webpack projects, be su
 
 ### Customize Vite config
 
-The builder will _not_ read your `vite.config.js` file by default.
+The builder _will_ read your `vite.config.js` file, though it may change some of the options in order to work correctly.
 
-In `.storybook/main.js` (or whatever your Storybook config file is named)
-you can override the Vite config:
+In `.storybook/main.js` (or whatever your Storybook config file is named), you can override the merged Vite config:
 
 ```javascript
 // use `mergeConfig` to recursively merge Vite options
-const { mergeConfig } = require('vite');
+import { mergeConfig } from 'vite';
 
-module.exports = {
+const config: StorybookConfig = {
```

Comment:
Did we want to make this example typescript? If not we should remove the type annotation